types/firestore.d.ts

```diff
@@ -1895,6 +1916,38 @@ declare namespace FirebaseFirestore {
     valueOf(): string;
   }
 
+  /**
+   * Builds a Firestore data bundle with results from the given document and query snapshots.
+   */
+  export class BundleBuilder {
+    /** The ID of this bundle. */
+    readonly bundleId: string;
+
+    /**
+     * Adds a Firestore document snapshot or query snapshot to the bundle.
+     * Both the documents data and the query read time will be included in the bundle.
+     *
+     * @param documentSnapshot A document snapshot to add.
+     * @returns This instance.
+     */
+    add(documentSnapshot: DocumentSnapshot): BundleBuilder;
+
+    /**
+     * Adds a Firestore document snapshot or query snapshot to the bundle.
+     * Both the documents data and the query read time will be included in the bundle.
```

Comment:
Please update to only refer to QuerySnapshots.

Reply:
Done.